src/Sarif.Multitool/ResultMatchingTestOptions.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using CommandLine;
+using Microsoft.CodeAnalysis.Sarif.Driver;
+
+namespace Microsoft.CodeAnalysis.Sarif.Multitool
+{
+    [Verb("result-matching-test", HelpText = "Match Results across runs in the same Group in a folder.")]
```

Comment:
>Verb [](start = 5, length = 4)

I believe the Verb attribute has a property named Hidden prevents the verb from being displayed in the command line help, while still being callable.

Reply:
After more thought, I think we might want this mode for real use, so I renamed it "match-result-set" and am going to leave it visible.

Reply:
[ex: Get Clean scan comparisons over commit history]